src/renderer.c

```diff
@@ -56,7 +56,10 @@ typedef struct RenFont {
   ERenFontHinting hinting;
   unsigned char style;
   unsigned short underline_thickness;
+#ifdef _WIN32
   unsigned char *file;
+  HANDLE file_handle;
```

Comment:
Doesn't seems to be used.

Reply:
it's intentional. I opened the file with shared write access so that windows will disallow delete and writes as long as the font is used (which is the same behavior with fopen and freetype).

Reply:
ahh, for a moment I read the code without seeing it belonged to the RenFont struct hehe, ok this looks good